app/presenters/finder_content_item_presenter.rb

```diff
@@ -48,6 +48,7 @@ def details
       format_name: metadata.fetch("format_name", nil),
       signup_link: metadata.fetch("signup_link", nil),
       show_summaries: metadata.fetch("show_summaries", false),
+      summary: metadata.fetch("summary", nil),
```

Comment:
Can you do this in a separate commit? Maybe even in a separate PR? It's not related to the ESI funds specifically.